tests/test_sails.py

```diff
@@ -0,0 +1,231 @@
+import pytest
+import requests
+import json
+from pymongo import MongoClient
+
+
+"""
+Basic Queries
+"""
+
+query = {
+    'firstName' : 'test_use',
+    'lastName' : 'test01',
+    'email' : 'test@capstonedd.com',
+    'type' : 'buyer'
+  }
+  
+findq = {
+    'firstName' : 'test_use'
+  }
+  
+testq = {
+    'lastName' : 'test02'
+  }
+  
+dataquery = json.dumps(query)
+datafquery = json.dumps(findq)
```

Comment:
We probably want to implement some kind of setup/teardown for each test. https://pytest.org/latest/xunit_setup.html